firefox_desktop/explores/crashes.explore.lkml

```diff
@@ -0,0 +1,39 @@
+include: "//looker-hub/firefox_desktop/views/crashes_daily.view.lkml"
+include: "../views/crash_usage.view.lkml"
+include: "/shared/views/countries.view.lkml"
+
+
+explore: crash_usage {
+  label: "Crashes"
+  description: "Crash counts for Desktop Firefox, derived from the crash ping."
```

Comment:
nit: even though the explore and view have the same name, I always like to add the `from`/`view_name` parameter so it's 100% clear